src/components/network-chart/index.js

```diff
@@ -95,6 +106,17 @@ class NetworkChartImpl extends React.PureComponent<Props> {
     );
   };
 
+  scrollSelectionIntoView() {
+    const { selectedNetworkMarkerIndex, markerIndexes } = this.props;
+    if (this._virtualListRef.current && selectedNetworkMarkerIndex !== null) {
+      const list = this._virtualListRef.current; // this temp variable so that flow knows that it's non-null
```

Comment:
nit: if you define the const before the `if` block, you can use it in the `if` condition and make it somewhat shorter.